### PR TITLE
 GEODE-4728 - Update defaultSystem/geode.properties

### PR DESCRIPTION
**Pull request by moleske**
@davebarnes97 and I stumbled upon these when looking at some documentation

### CODE REVIEW

File: defaultSystem/geode.properties [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# Default Gemfire C++ distributed system properties
+# Default C++ distributed system properties
```
Comment (pivotal-jbarrett) [inline defaultSystem/geode.properties]: These properties are also loaded by the .NET client. I wonder if we can give them a better name.

File: defaultSystem/geode.properties [hunk 2]
```diff
@@ -22,13 +22,13 @@
...
 #
-## Cache region configurtion
+## Cache region configuration
 #
 #cache-xml-file=cache.xml
 #
 ## Log file config
 #
-#log-file=gemfire_cpp.log
+#log-file=geode_cpp.log
```
Comment (pivotal-jbarrett) [inline defaultSystem/geode.properties]: Same here.

File: defaultSystem/geode.properties (hunk 1 above)
Comment (pivotal-jbarrett) [reply defaultSystem/geode.properties]: Geode Native probably. This file shouldn't get confused with the one for the Java client/server.

**Comment by pivotal-jbarrett** [comment]
@moleske please use the GEODE number in the PR title and at least the first commit message.

**Comment by moleske** [comment]
My bad, was on GitHub when we saw it and forgot to check Jira before blindly updating comments

**Pull request merged**